spec/input/_dev/spec.yml

```
@@ -0,0 +1,19 @@
+spec:
+  additionalContents: false
+  developmentFolder: true
+  contents:
+    - description: Folder containing resources related to building the package.
+      type: folder
+      name: build
+      required: false
+      $ref: "../../integration/_dev/build/spec.yml"
+    - description: Folder containing configuration related to deploying the package's integration service(s)
```

Should this description make reference to input instead of integration ? 

```suggestion
    - description: Folder containing configuration related to deploying the package's input service(s)
```

---

Yep, good catch, thanks.

---

Fixed.